libs/community/langchain_community/document_loaders/unstructured.py

```diff
@@ -96,18 +66,20 @@ def lazy_load(self) -> Iterator[Document]:
                     metadata.update(element.metadata.to_dict())
                 if hasattr(element, "category"):
                     metadata["category"] = element.category
+                if element.to_dict().get("element_id"):
+                    metadata["element_id"] = element.to_dict().get("element_id")
                 yield Document(page_content=str(element), metadata=metadata)
         elif self.mode == "paged":
-            text_dict: Dict[int, str] = {}
-            meta_dict: Dict[int, Dict] = {}
+            text_dict: dict[int, str] = {}
```

Comment:
Why this change?

Reply:
This stemmed from the this thread [here](https://unstructuredw-kbe4326.slack.com/archives/C044N0YV08G/p1718187499818419)